omnibridge/main.py

```diff
@@ -23,25 +25,19 @@ def run() -> int:
     list_parser = subparsers.add_parser("list", help="List keys/models/flows/wrappers")
     list_subparsers = list_parser.add_subparsers(help="list sub-command help", dest="object_to_list")
 
+    run_parser = subparsers.add_parser("run", help="Run model/flow")
+    run_subparsers = run_parser.add_subparsers(help="run sub-command help", dest="object_to_run")
+
     add_create_arguments(create_subparsers)
```

Comment:
You mean dynamically load arguments based on the command the user specifies?
If yes, then I don't think that would work, since you do want to have a detailed --help section with all the available arguments